service/etc/agama.yaml

```diff
@@ -20,6 +20,88 @@ web:
   ssl_cert: null
   ssl_key: null
 
+ALP-Dolomite:
+  software:
+    installation_repositories:
+      - url: https://updates.suse.com/SUSE/Products/ALP-Dolomite/1.0/x86_64/product/
+        archs: x86_64
+      - url: https://updates.suse.com/SUSE/Products/ALP-Dolomite/1.0/aarch64/product/
+        archs: aarch64
+      - url: https://updates.suse.com/SUSE/Products/ALP-Dolomite/1.0/s390x/product/
+        archs: s390
+      - url: https://updates.suse.com/SUSE/Products/ALP-Dolomite/1.0/ppc64le/product/
+        archs: ppc
+
+    mandatory_patterns:
+      - alp_base_zypper
+      - alp_cockpit
+      - alp_hardware
+    optional_patterns: null # no optional pattern shared
+    mandatory_packages:
+      - package: device-mapper # Apparently needed if devices at /dev/mapper are used at boot (eg. FDE)
+      - package: fde-tools # Needed for FDE with TPM, hardcoded here temporarily
+        archs: aarch64, x86_64
+      - package: libtss2-tcti-device0 # Same than fde-tools
+      - package: ppc64-diag # Needed for hardware-based installations
+        archs: ppc64
+    optional_packages: null
+    base_product: ALP-Dolomite
+
+  security:
+    tpm_luks_open: true
+    lsm: selinux
+    available_lsms:
+      # apparmor:
+      #   patterns:
+      #     - apparmor
+      selinux:
+        patterns:
+          - alp_selinux
+        policy: enforcing
+      none:
+        patterns: null
+
+  storage:
+    encryption:
+      method: luks2
+      pbkd_function: pbkdf2
+    volumes:
+      - "/"
+    volume_templates:
+      - mount_path: "/"
+        filesystem: btrfs
+        btrfs:
+          snapshots: true
+          read_only_root: true
```

Comment:
I think the code expects this to be `read_only` instead of `read_only_root`. I'm not sure which one is better, but it should be consistent. Either fix it here, either adapt VolumeTemplatesBuilder.